esp/src/src/nls/es/hpcc.ts

```diff
@@ -468,7 +475,23 @@ export = {
     Log: "Registro",
     LogAccessType: "Tipo de acceso al registro",
     LogDirectory: "Directorio de logs",
+    LogEventType: 'Tipo de Evento de Registro',
     LogFile: "Archivo de registro",
+    LogFilterComponentsFilterTooltip: 'Incluir solo registros del componente/contenedor seleccionado',
+    LogFilterCustomColumnsTooltip: 'Incluir las columnas de registro especificadas',
+    LogFilterEndDateTooltip: 'Incluir solo entradas hasta el final del rango de tiempo especificado',
+    LogFilterEventTypeTooltip: 'Incluir solo entradas para el tipo de evento de registro especificado',
+    LogFilterFormatTooltip: 'Formato de archivo de registro: CSV, JSON o XML',
+    LogFilterLineLimitTooltip: 'El número de líneas de registro que se incluirán',
+    LogFilterLineStartFromTooltip: 'Incluir líneas de registro a partir de este número de línea',
+    LogFilterRelativeTimeRangeTooltip: 'Un rango de tiempo que rodea el tiempo de la WU, +/- (en segundos)',
+    LogFilters: 'Filtros de Registro',
+    LogFilterSelectColumnModeTooltip: 'Especificar qué columnas incluir en el archivo de registro',
+    LogFilterSortByTooltip: 'ASC - el más antiguo primero, DESC - el más nuevo primero',
+    LogFilterStartDateTooltip: 'Incluir líneas de registro a partir del inicio del rango de tiempo especificado',
+    LogFilterTimeRequired: 'Elija entre "Desde y Hasta la Fecha" o "Búfer de Tiempo Relativo de Registro"',
```

Comment:
Confirm the quotes used here are correct

Reply:
Corrected all the quotes that needed to be adjusted. Thanks for the catch.

Reply:
I still see single quotes encapsulating the string and unescaped double quotes w/in the string. I don't know enough about the syntax, but that seems to be different than the other strings.